File: src/main/java/com/swp391/koibe/repositories/FeedBackRepository.java
package com.swp391.koibe.repositories;

import com.swp391.koibe.models.Feedback;
import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;

public interface FeedBackRepository extends JpaRepository<Feedback, Long> {

    Boolean existsByUserIdAndOrderId(Long userId, Long orderId);
    Optional<Feedback> findByOrderId(Long orderId);

}

File: src/main/java/com/swp391/koibe/repositories/FeedBackRepository.kt
package com.swp391.koibe.repositories

import com.swp391.koibe.models.Feedback
import org.springframework.data.jpa.repository.JpaRepository

interface FeedBackRepository : JpaRepository<Feedback, Long> {
    fun existsByUserIdAndOrderId(userId: Long, orderId: Long): Boolean
}
